add_mq0_and_filter.py

```diff
@@ -0,0 +1,116 @@
+import argparse
+import sys
+import vcfpy
+import csv
+from collections import OrderedDict
+
+def to_array(dictionary):
+    array = []
+    for key, value in dictionary.items():
+        array.append("{}|{}".format(key, value))
+    return sorted(array)
+
+def parse_tsv_file(args):
+    values={}
+    with open(args.values_file,'r') as tsvin:
+        tsvin = csv.reader(tsvin, delimiter='\t')
+        for row in tsvin:
+            if any(x.strip() for x in row): #skip blank lines
+                values[(row[0] + ":" + row[1])] = row[2]
+    return values
+
+
+def create_vcf_reader(args):
+    vcf_reader = vcfpy.Reader.from_path(args.input_vcf)
+    if 'MQ0' in vcf_reader.header.format_ids():
+        print("FORMAT already contains a MQ0 field. It's value will be overwritten when matches are found")
+    if 'MQ0FRAC' in vcf_reader.header.format_ids() :
+        print("FORMAT already contains a MQ0FRAC field. It's value will be overwritten when matches are found")
+    return vcf_reader
+
+def create_vcf_writer(args, vcf_reader):
+    if args.output_vcf:
+        output_file = args.output_vcf
+    else:
+        (head, sep, tail) = args.input_vcf.rpartition('.vcf')
+        output_file = ('').join([head, '.info.vcf', tail])
+
+    new_header = vcf_reader.header.copy()
+
+    if not 'MQ0' in vcf_reader.header.format_ids():
+        od = OrderedDict([('ID', 'MQ0'), ('Number', '1'), ('Type', 'Float'), ('Description', 'Number of MAPQ == 0 reads covering this site in the tumor')])
+        new_header.add_format_line(od)
+    if not 'MQ0FRAC' in vcf_reader.header.format_ids():    
+        od2 = OrderedDict([('ID', 'MQ0FRAC'), ('Number', '1'), ('Type', 'Float'), ('Description', 'Fraction of MAPQ == 0 reads covering this site in the tumor')])
+        new_header.add_format_line(od2)
+    if not 'MQ0FRAC' in vcf_reader.header.filter_ids():    
```

Comment:
Is this one meant to check `MAPQ0` (matching the ID added below)?
```suggestion
    if not 'MAPQ0' in vcf_reader.header.filter_ids():    
```